sb.web.app/src/components/Members/userProfile.vue

```diff
@@ -95,12 +95,12 @@ export default {
       this.comment = this.$refs.commentInput.getInput()
       if (this.tkn && Number.isInteger(Number(this.tkn)) && Number(this.tkn) > 0) {
         const saldo = await getAvailableBalance()
-        if (saldo < this.tkn) {
+        if (saldo.totalAvailableBalance < this.tkn) {
           this.notEnoughBkrMsg = true
         } else {
           const userSaldo = await getUserAvailableBalance(this.profileData.accountName)
           const userLimits = await getUserLimits(this.profileData.accountName)
-          if (userSaldo + userLimits.min + Number(this.tkn) > userLimits.max) {
+          if (userSaldo.totalAvailableBalance + userLimits.min + Number(this.tkn) > userLimits.max) {
```

Comment:
I think we should rename this.tkn - its a very confusing name given that org.tkn is a string. 

Also if its a Number why can't we convert it to a number when we do 
 this.tkn = Number(this.$refs.tknInput.getInput()) ?

Reply:
i Agree we should change the naming then deal with the above comment